packages/models/src/interviews.ts

```diff
@@ -165,6 +191,7 @@ export class Interviews {
       baseBuilder.whereIn(this.column("signer"), signers);
 
     if (!isEmpty(rules)) baseBuilder.whereIn(this.column("rule_id"), rules);
```

Comment:
Empty line. Make use of empty lines to group differt logic-group togther. This will make it easier to read.  

```suggestion
    if (!isEmpty(rules)) baseBuilder.whereIn(this.column("rule_id"), rules);

```